src/Middleware.js

```diff
@@ -0,0 +1,63 @@
+import {KEYFRAMES, DECLARATION} from './Enum.js'
```

Comment:
If I fix errors in this file (mainly just import missing stuff) all existing tests pass - should I commit it and push out? Or are you in the middle of some work?

Reply:
Middleware.js and Serialize.js are going to change in the next commit.